platform/client-admin/src/forms/create-volunteer.ts

```diff
@@ -0,0 +1,61 @@
+import SimpleSchema2Bridge from 'uniforms-bridge-simple-schema-2';
+import SimpleSchema from 'simpl-schema';
+
+const volunteerForm = new SimpleSchema({
+    firstName: {
+        type: String,
+        max: 120
+    },
+    lastName: {
+        type: String,
+        max: 120
+    },
+    email: {
+        type: String,
+        max: 220
+    },
+    username: {
+        type: String,
+        max: 220
+    },
+    address1: {
+        type: String,
+        max: 400
+    },
+    address2: {
+        type: String,
+        max: 400
+    },
+    city: {
+        type: String,
+        required: false,
+        max: 100
+    },
+    postcode: {
+        type: Number,
+        required: false,
+    },
+    dateOfBirth: {
+        type: Date,
+        required: false,
+        uniforms: {
+            label: "Date Of Birth"
+        },
+    },
+    canPhoneCall: {
```

Comment:
I thought we decided to refine this and remove?

Reply:
Good catch. Should be removed. I forgot to remove it.